tests/integration-larvavel.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Inspired by https://github.com/nunomaduro/larastan/blob/669b489e10558bd45fafc2429068fd4a73843802/tests/laravel-test.sh
+#
+# Create a fresh Laravel installation, install our package in it and run some
+# basic tests to ensure everything works.
+#
+# This script is meant to be run on Travis CI
+
+echo "Prevent shallow repository error"
+git fetch --unshallow
```

Comment:
This refers to this https://travis-ci.org/rebing/graphql-laravel/jobs/548915299#L621-L637

One has to see it to understand it, I guess. It only happened for the "PR" build, not for branch builds.